client/landing/stepper/declarative-flow/internals/steps-repository/importer-wordpress/index.tsx

```diff
@@ -28,7 +28,11 @@ const ImporterWordpress: Step = function ( props ) {
 		<>
 			{ showMigrationModal && (
 				<MigrationAssistanceModal
-					onConfirm={ () => stepNavigator?.goToCheckoutPage?.( WPImportOption.EVERYTHING ) }
+					onConfirm={ () => {
+						stepNavigator?.goToCheckoutPage?.( WPImportOption.EVERYTHING, {
+							redirect_to: `/setup/${ props.flow }/migrateMessage?siteSlug=${ siteSlug }`,
```

Comment:
I would escape both of these. 
```suggestion
							redirect_to: `/setup/${ encodeURIComponent ( props.flow ) }/migrateMessage?siteSlug=${ encodeURIComponent( siteSlug ) }`,
```